### ISSUE TITLE
clang: djinni_support.cpp:485: error: unused parameter

### ISSUE THREAD

**Issue by alexcohn** [dropbox/djinni#294]
Having switched to Android NDK r14 and clang, we now see 

```
[armeabi-v7a] Compile++ thumb: foundation <= djinni_support.cpp
djinni/support-lib/jni/djinni_support.cpp:485:57: error: unused parameter 'data' [-Werror,-Wunused-parameter]
static std::wstring implUTF16ToWString(const char16_t * data, size_t length)
                                                        ^
djinni/support-lib/jni/djinni_support.cpp:485:70: error: unused parameter 'length' [-Werror,-Wunused-parameter]
static std::wstring implUTF16ToWString(const char16_t * data, size_t length)

```

Is there any compiler that won' be happy with hiding the unused parameters?

`static std::wstring implUTF16ToWString(const char16_t * /*data*/, size_t /*length*/)
`

**Comment by j4cbo** [comment]
`(const char16_t * /*data*/, size_t /*length*/)` is the right fix - can you send a pull request?

**Comment by artwyman** [comment]
Resolved in #296
